tribe-ext-view-tickets/index.php

```diff
@@ -75,10 +128,11 @@ public function show_ticket_page() {
 
 		// Include the class and functions that power this.
 		require_once( 'src/functions/general.php' );
-		require_once( 'src/Tribe/Extension/Tickets_Order_Helper.php' );
+		require_once( 'src/Tribe/Extension/Tickets_Order_Helper2.php' );
```

Comment:
Curious - why name the file `Tickets_Order_Helper2` if there's no `Tickets_Order_Helper`: seems like, since we're including it via a `require_once()`, there isn't much or any potential for a conflict?